tests/providers/kubernetes/test_base.py

```diff
@@ -75,16 +78,17 @@ def testClusterListNamespacedPods(self, mock_k8s_api):
     mock_k8s_api_func = mock_k8s_api.return_value.list_namespaced_pod
     mock_k8s_api_func.return_value = mock_pods
 
-    pods = base.K8sCluster(api_client=k8s_mocks.MOCK_API_CLIENT).ListPods(
-      mock_namespace
-    )
+    pods = cluster.K8sCluster(
+        api_client=k8s_mocks.MOCK_API_CLIENT).ListPods(mock_namespace)
 
     # Assert API and corresponding function was called appropriately
     mock_k8s_api.assert_called_with(k8s_mocks.MOCK_API_CLIENT)
     mock_k8s_api_func.assert_called_with(mock_namespace)
     # Assert returned pods correspond to provided response
-    self.assertTrue(set(pod.name for pod in pods),
-                    set(pod.metadata.name for pod in mock_pods.items))
+    self.assertTrue(
```

Comment:
Isn't this supposed to be `assertEqual()`?

Reply:
Thanks for spotting that